cluster_tools/cluster_tools/schedulers/cluster_executor.py

```diff
@@ -525,50 +581,57 @@ def shutdown(self, wait=True):
                 pass
         self.files_to_clean_up = []
 
-    def map(self, func, args, timeout=None, chunksize=None):
+    # TODO: args should be *iterables, this would be a breaking change!
```

Comment:
todo. whats your plan here? :)

Reply:
This is part of the follow-up points mentioned in the description of this PR. I'd still leave the TODO in the code if you agree.